esi_leap/objects/offer.py

```diff
@@ -122,9 +121,9 @@ def to_marketplace_dict(self):
         offer_dict['end_time'] = offer_dict.pop('end_date').isoformat()
         offer_dict['cost'] = offer_dict['properties'].get('floor_price', 0)
         offer_dict['server_id'] = offer_dict.pop('resource_uuid')
-        offer_dict['provider_id'] = offer_dict.pop('uuid')
-        offer_dict['creator_id'] = offer_dict.pop('project_id')
-
+        offer_dict['provider_offer_id'] = offer_dict.pop('uuid')
+        offer_dict['creator_id'] = offer_dict['project_id']
```

Comment:
The `creator_id` is no longer necessary when creating offers in the marketplace

Reply:
Okay, I've removed it.